packages/media-sound/cmus/cmus.exlib

```diff
@@ -105,6 +106,11 @@ cmus_src_configure() {
     local s
     local myconf=(
         $(for s in "${SRC_CONFIGURE_OPTIONS[@]}"; do option_yesno ${s}; done)
+        $(if option providers:elogind || option providers:systemd; then
```

Comment:
You don't need this if branch, you can still pass "systemd mpris" (via `option_yesno`) to `option`, you just need to add the `providers:` prefix before the option name.